dev-python/foolscap/foolscap-21.7.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{8,9} )
+
+inherit distutils-r1
+
+DESCRIPTION="RPC protocol for Twisted"
+HOMEPAGE="https://github.com/warner/foolscap http://foolscap.lothar.com/trac https://pypi.org/project/foolscap/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="doc +ssl test"
+
+# Many tests still failing (#657604), others rely on network
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/six[${PYTHON_USEDEP}]
+	>=dev-python/twisted-16[${PYTHON_USEDEP}]
+	ssl? ( dev-python/pyopenssl[${PYTHON_USEDEP}] )
+"
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
I don't think this part is needed, but check the default behaviour of the eclass